utilities/shared_disk.py

```
@@ -308,3 +308,242 @@ def verify_shared_disk_data(
+            func=_try_get_drive_letter,
+        ):
+            if sample and len(sample) == 1:
+                LOGGER.info(f"{vm_label}: SHARED volume is drive {sample}:")
+                return sample
+    except TimeoutExpiredError as exc:
+        raise GuestCommandError(
+            f"{vm_label}: Volume with label '{_SHARED_VOLUME_LABEL}' not found after "
+            f"{_WIN_VOLUME_DISCOVERY_TIMEOUT}s. Ensure the shared disk on the source VM "
+            f"has an NTFS volume labeled '{_SHARED_VOLUME_LABEL}'."
+        ) from exc
+    raise GuestCommandError(f"{vm_label}: SHARED volume not found")
```

Ruff (0.15.17)

[warning] Avoid specifying long messages outside the exception class

(TRY003)